apartmentsbot/bot_runner.py

```diff
@@ -0,0 +1,79 @@
+from telebot.states.sync.middleware import StateMiddleware
+from telebot import custom_filters, types
+from telebot.states.sync.context import StateContext
+from apartmentsbot.apartment_manager import state_storage, MyStates
+from apartmentsbot.bot_handler import BotHandler
+import logging
+import telebot
+import configparser
+
+logging.basicConfig(level=logging.INFO, filename="apartment_bot.log", filemode="w",
+                    format="%(asctime)s %(levelname)s %(message)s")
+
+config = configparser.ConfigParser()
+config.read("config.ini")
```

Comment:
just don't use .ini files🥲 Use yaml as a config file or json also works fine.

Reply:
I mean you can leave it of course but for the next time you'd better to choose another format

Reply:
done